bundle/permissions/apply_folder_permissions.go

```diff
@@ -57,7 +57,7 @@ func giveAccessForWorkspaceRoot(ctx context.Context, b *bundle.Bundle) error {
 		return err
 	}
 
-	_, err = w.UpdatePermissions(ctx, workspace.WorkspaceObjectPermissionsRequest{
+	_, err = w.SetPermissions(ctx, workspace.WorkspaceObjectPermissionsRequest{
```

Comment:
Note to self: this changes the existing behavior; could be something to split out into a separate PR.